src/command_modules/azure-cli-resource/azure/cli/command_modules/resource/custom.py

```diff
@@ -67,6 +73,124 @@ def create_resource_group(rg_name, location, tags=None):
     return rcf.resource_groups.create_or_update(rg_name, parameters)
 
 
+def create_appliance(resource_group_name,  # pylint: disable=too-many-arguments
+                     appliance_name, managedby_resource_group_id,
+                     location, kind, managedapp_definition_id=None,
+                     plan_name=None, plan_publisher=None, plan_product=None,
+                     plan_version=None, tags=None, parameters=None):
+    ''' Create a new managed application.
+    :param str resource_group_name:the desired resource group name
+    :param str appliance_name:the managed application name
+    :param str kind:the managed application kind. can be marketplace or servicecatalog
+    :param str plan_name:the managed application package plan name
+    :param str plan_publisher:the managed application package plan publisher
+    :param str plan_product:the managed application package plan product
+    :param str plan_version:the managed application package plan version
+    :param str tags:tags in 'a=b c' format
+    '''
```

Comment:
According to [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings) use triple-double-quote here.